### DISCUSSION TITLE
Traction-free vs Stress-free boundary

### DISCUSSION THREAD

**Discussion by zzzmx-josh** [idaholab/moose#29535] [Q&A Modules: Solid mechanics] [answered]
I understand that when no displacement is prescribed on the boundary (i.e., free boundary conditions), the traction-free condition is defined as:

$$\sigma_{ij} \cdot n_j = 0 \quad$$,

which implies no normal or tangential traction on the boundary. However, in my simulation, I aim to implement stress-free boundary conditions, where all components of the stress tensor vanish on the boundary:  
$$\sigma_{ij} = 0$$.

Could you please advise on how to achieve this in MOOSE? Are there specific settings, constraints, or additional steps required to enforce $$\sigma_{ij} = 0$$ on the boundary?

Thank you for your time and assistance.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Stress free boundary should be the default boundary condition when no BC is applied on a boundary
You should be able to verify that by exporting the stress material property on a boundary to an auxiliary variable (defined in the whole volume by default, but the auxkernel would be defined only on the boundary)

**Reply by zzzmx-josh** [reply]
I’ve now figured it out, Thank you so much for your help!
